src/openassetio-core/src/hostApi/ManagerFactory.cpp

```diff
@@ -91,6 +91,14 @@ ManagerPtr ManagerFactory::defaultManagerForInterface(
     return nullptr;
   }
 
+  return defaultManagerForInterface(configPath, hostInterface, managerImplementationFactory,
+                                    logger);
+}
+
+ManagerPtr ManagerFactory::defaultManagerForInterface(
+    const std::string_view configPath, const HostInterfacePtr& hostInterface,
+    const ManagerImplementationFactoryInterfacePtr& managerImplementationFactory,
+    const log::LoggerInterfacePtr& logger) {
```

Comment:
[fix] Can't anchor a comment below, but we need to remove:

```
    msg += "' [" + kDefaultManagerConfigEnvVarName + "]";
```
As its no longer always from the env var. Maybe add something to the env var method to indicate that it was used?

Reply:
Whoops. Fixed and tweaked the logging in 08f2b1f0